pkg/controller/egress/store/egressgroup_test.go

```diff
@@ -0,0 +1,183 @@
+package store
```

Comment:
license header, "Check tidy, code generation and manifest" is failing.

Reply:
Done.